contrib/ffmpeg/A00-qsv.patch

```diff
@@ -1565,6 +1556,7 @@ index 0000000..53a9f16
 +    if (!qsv)
 +        return AVERROR(ENOMEM);
 +
++    qsv_decode = qsv->dec_space;
```

Comment:
Given that it's hard to read diffs of diffs, this is the line I added to fix the problem.